src/utils/whiteSpace.js

```diff
@@ -0,0 +1,8 @@
+import { style } from 'styled-system';
+
+const whiteSpace = style({
+  prop: 'whiteSpace',
+  cssProperty: 'whiteSpace'
```

Comment:
I guess the cssProperty whiteSpace gets dash-cased internally?

Reply:
That's correct!